HabboHotel/Groups/Forums/GroupForumManager.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Dapper;
+using Plus.Database;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Plus.HabboHotel.Groups.Forums
+{
+    public class GroupForumManager : IGroupForumManager
+    {
+        private readonly IDatabase _database;
+        readonly List<GroupForum> Forums;
```

Comment:
I would make this a `ConcurrentDictionary<int, GroupForum>` for easier lookup. Otherwise every call to `TryGetForum` has to iterate the full list and is also not thread safe.

You're missing the `private` access modifier here.